webapp-client/package.json

```diff
@@ -0,0 +1,34 @@
+{
+  "name": "webapp-client",
+  "version": "0.1.0",
+  "private": true,
+  "dependencies": {
+    "@testing-library/jest-dom": "^4.2.4",
+    "@testing-library/react": "^9.3.2",
+    "@testing-library/user-event": "^7.1.2",
```

Comment:
These can go in devDependencies and I think react-scripts too

Reply:
Changed for further review in Cynthia's PR